src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -120,6 +120,7 @@ func (client *MobileClient) updateConfig() error {
 	var buf []byte
 	var err error
 	if buf, err = pullConfigFile(client.fronter); err != nil {
+		log.Fatalf("Could not update config: '%v'", err)
```

Comment:
@myleshorton Just to make sure, `log.Fatalf` will quit the program, is it intended? `MobileClient.pollConfiguration()` will call this method over and over again.